models/pendo__guide.sql

```diff
@@ -0,0 +1,36 @@
+{{ config(enabled=false) }}
```

Comment:
Shouldn't this instead be tied to a variable?

Reply:
Or are we planning to enable this by default

Reply:
i just set this to false while developing, since we paused on the guide stuff